main.md

```diff
@@ -395,11 +504,9 @@ be disclosed in full.
 {
   "iss": "https://example.com/issuer",
   "cnf": {
-    "jwk" : {
-        "kty": "RSA",
-        "n": "pm4bOHBg-oYhAyPWzR56AWX3rUIXp11_ICDkGgS6W3ZWLts-hzwI3x65659kg4hVo9dbGoCJE3ZGF_eaetE30UhBUEgpGwrDrQiJ9zqprmcFfr3qvvkGjtth8Zgl1eM2bJcOwE7PCBHWTKWYs152R7g6Jg2OVph-a8rq-q79MhKG5QoW_mTz10QT_6H4c7PjWG1fjh8hpWNnbP_pv6d1zSwZfc5fl6yVRL0DV0V3lGHKe2Wqf_eNGjBrBLVklDTk8-stX_MWLcR-EGmXAOv0UBWitS_dXJKJu-vXJyw14nHSGuxTIK2hx1pttMft9CsvqimXKeDTU14qQL1eE7ihcw",
-        "e": "AQAB"
-    }
+    "kty": "RSA",
```

Comment:
looks like Giuseppe's PR on adding `jwk` top level claim to `cnf` has been overwritten?

Reply:
As above, not overwritten but not in this branch yet.